### DISCUSSION TITLE
Series dependent content not represented in maps

### DISCUSSION THREAD

**Discussion by MoBosse** [open-sdg/open-sdg#1717] [General]
I just noticed that some series and unit dependent items are not present in the map:

- Titles 
  - Title of map sticks to the title, set for the first series when using `graph_titles`
  - The `indicator_name` is shown above the map when having no `graph_titles` (although `graph_title_from_series` in the site config is set to true)

-  Footnotes
   - The content of the `footer_fields` disappears compleatly when changing to the map view
   - `data_footnote` is shown under the map but is obviously not series dependent 

- Units (when there is only one unit available)
  - The content of `computation_units` disappears completely from the footer when switching to map view
  - The only way to display the unit seems to be to have a Units column in the data. The downside here is that it looks like you have the option to change it, since it is displayed with a radio button when you click the "Change Brakedowns" button. 

- Precision
   - has no impact either. Actually I would rather not have decimal places in the maps legend, but a) I wonder how the legend would look for e.g. values within a range from 0 to 1 and b) I think the tooltip values cound get decimal places if they are defined for the selected series

There are probably other elements that are relevant to maps, but these are the ones I've noticed so far.

**Comment by christoph-LSN** [comment]
@MoBosse @brockfanning 

I have noticed this too:

example indicator: https://christoph-lsn.github.io/MT_Site/1-3-1/

only "%" is shown but not "in 1000"

![image](https://user-images.githubusercontent.com/94443931/177174111-2197b75a-548d-43ad-b2e9-468e9ec58280.png)

![image](https://user-images.githubusercontent.com/94443931/177174259-803c98f9-b31b-492d-b394-ee55e24ccac3.png)

**Comment by brockfanning** [comment]
I believe these should be fixed in the next release (2.1.0) so I will mark this as answered (but feel free to update if I'm wrong there). EDIT - Oops, I thought I could do that. Anyway, just noting that these should be fixed.
